lib/screens/search.dart

```diff
@@ -0,0 +1,35 @@
+import 'package:flutter/material.dart';
+import 'package:google_fonts/google_fonts.dart';
+
+class mysearch extends StatelessWidget {
+  const mysearch({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      backgroundColor: Colors.black,
+      body: Column(
+      crossAxisAlignment: CrossAxisAlignment.center,
+      children: [
+        const SizedBox(height: 10,),
+        Padding(
+          padding: const EdgeInsets.only(left: 20.0),
```

Comment:
try replacing it with this instead it will look better
padding: const EdgeInsets.only(
                  top: 50, bottom: 20, left: 20, right: 20),
try replacing it with this instead it will look better